packages/core/hooks/useFileAccessContextMenu.ts

```diff
@@ -68,6 +78,26 @@ export default (filters?: FileFilter[], onDismiss?: () => void) => {
                         items: openWithSubMenuItems,
                     },
                 },
+                ...(isQueryingAicsFms && !isOnWeb
+                    ? [
+                          {
+                              key: "go-to-file-location",
+                              text: "Go to file location",
+                              title: "Show selected file in your native file browser",
+                              disabled:
```

Comment:
For things like this where they can only apply to one item we just use the focused item (of which there has to be one by default) so it would make sense to me to remove the restriction on `fileSelection.count() > 1`. This _should_ be disabled or removed from the list of options when rendering on a selected folder though.
`disabled: isNil(fileDetails?.localPath)` should cover you

Reply:
That's waaaay easier. Done in 866d43b759a52c0e1da9297b489aa2afec56c31f